### ISSUE TITLE
Talking about the Markup

### ISSUE THREAD

**Issue by Grienauer** [betterembed/neos-embed#8]
would be great to get markup adoptions and Improvements communicated and also moved to https://codepen.io/Grienauer/pen/NWPaKBd
I think I will create a github project for it too, so we can discuss/adopt/merge better!

Please let's discuss:
https://github.com/betterembed/neos-betterembed/blob/master/Resources/Private/Fusion/Component/Default/Default.fusion

**Questions:**
<h1>{props.title}</h1>
I think it would be better to use a h2 or even h3
due to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article it is possible, but as you can see, also in the demo it is a h2

alt text
✅ "Thumbnail from the embeded URL" is a good option.

Extra wrapper and different markup
I don't know, why we need a top area or multiple spanm elements inside of the button element.
I think this is also a matter of taste.
I think we should use the same markup and work on one "official" base or completelz split it and implement it how it should be best for each system.
I am a fan of as less divs as possible and to use just js-* classnames if a JavaScript should listen/change something.

Thx for your wort it is well build and looks fantastic!
Nico

**Comment by davidspiola** [comment]
yes, that was the intension. The adoptions are not final and need to be discussed. Thx for opening an issue
